trace-sdk/src/test/java/io/bitrise/trace/data/application/ApplicationVersionNameDataCollectorTest.java

```diff
@@ -0,0 +1,52 @@
+package io.bitrise.trace.data.application;
+
+import android.content.Context;
+
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import io.bitrise.trace.BuildConfig;
+import io.bitrise.trace.configuration.ConfigurationManager;
+import io.bitrise.trace.data.collector.application.ApplicationVersionNameDataCollector;
+import io.bitrise.trace.data.dto.Data;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Unit tests for {@link ApplicationVersionNameDataCollector}
+ */
+public class ApplicationVersionNameDataCollectorTest {
+
+    private final static String versionName = "1.2.3";
+    private final Context mockContext = Mockito.mock(Context.class);
+    private final ApplicationVersionNameDataCollector collector = new ApplicationVersionNameDataCollector(
+            mockContext);
+
+    @Test
+    public void collectData_configurationManagerInitialised() {
```

Comment:
Please see [this](https://github.com/bitrise-io/trace-android-sdk/pull/199/files#r635170192).

Reply:
Wrote the reply in that thread - to keep the conversation in one place.